packages/drivers/file-driver/src/fileDocumentDeltaConnection.ts

```diff
@@ -204,7 +204,7 @@ export class ReplayFileDeltaConnection
         debug("dropping the outbound signal and wait for response");
     }
 
-    public disconnect() {
+    public close() {
         debug("no implementation for disconnect...");
```

Comment:
```suggestion
        debug("no implementation for close...");
```

Reply:
Thanks! I'd rather remove all debug() statements altogether - I believe they are great for initial (local) debugging, but shipping code should use telemetry...

Reply:
Either seems fine to me, just want to avoid stale terminology